satellite/admin-back-office-v1/user-roles.drawio

```diff
@@ -0,0 +1,97 @@
+<mxfile host="Electron" modified="2023-08-09T11:37:20.511Z" agent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/21.6.5 Chrome/114.0.5735.243 Electron/25.3.1 Safari/537.36" etag="m65WNdth2jxJov2d9NVG" version="21.6.5" type="device">
```

Comment:
Tip, you should be able to use mermaid diagrams. The main benefit is that modifying the diagram inline in the markdown is more likely to stay up to date.

In this case you could create a subgraph per entity (e.g. Account, Project, Bucket) and then have nodes for each role and draw a line to them. Alternatively a C4 diagram could work.

Reply:
I would have loved to draw a mermaid one, but this kind of diagram cannot be done and I believe that this kind is clearer than one with boxes and arrows.

Reply:
That's cool, I will add markdown table.